src/AdminConsole/Components/Pages/Organization/SettingsComponents/SecurityComponent.razor

```diff
@@ -0,0 +1,25 @@
+@inject IDataService DataService
+@inject IAdminService AdminService
+
+<Panel Header="Security">
+    @if (SecurityForm != null)
+    {
+    <EditForm class="flex flex-col space-y-4" FormName="@SecurityFormName" Model="SecurityForm" OnValidSubmit="@OnSecurityFormSubmittedAsync">
+        <div class="flex flex-col space-y-4">
+            <div class="flex flex-col space-y-2">
```

Comment:
We should probably have some wording here about what exactly "Enable Magic Links" Is.

If an admin loses their passkey, they will be locked out of the AdminConsole if this is disabled.  This should be made explicitly clear.

Reply:
<img width="803" alt="image" src="https://github.com/user-attachments/assets/a5ef85c8-dd1b-4c6c-8028-0178a2390095">

Reply:
Magic links allow administrators to sign into the admin console without a password.
- When enabled, if an administrator does not have his or her passkey, they can sign in with a magic link sent to their email. However, if an administrator's email is compromised, the attacker can sign in as the administrator.
- When disabled, administrators must sign in with their passkeys. If their passkeys are lost, a different administrator must reset their passkeys.